internal/aws/xray/telemetry/sender.go

```diff
@@ -163,27 +161,20 @@ func (p envMetadataProvider) get() string {
 }
 
 type ec2MetadataProvider struct {
-	clientIMDSV2Only     *imds.Client
-	clientIMDSV1Fallback *imds.Client
-	metadataKey          string
-	logger               *zap.Logger
```

Comment:
@jefchien do we need this logger?

Reply:
I added this as part of the imds to log when it failed for imds v2 and v1 so I assume no